dev/breeze/src/airflow_breeze/commands/release_management_commands.py

```diff
@@ -446,26 +448,29 @@ def package_version(
     for pyproject_toml_path in pyproject_toml_paths:
         original_contents.append(pyproject_toml_path.read_text())
     update_version_in__init_py = False
+    base_package_version = original_package_version
     if version_suffix:
         if original_package_version.endswith(f".{version_suffix}"):
             get_console().print(
                 f"[info]The {original_package_version} already has suffix {version_suffix}. Not updating it.\n"
             )
-            package_version = original_package_version
+            base_package_version = original_package_version
         elif not release_version_matcher.match(original_package_version):
             get_console().print(
                 f"[warning]Normally you should only pass version suffix if package version "
                 f"does not have suffix in code (it is  {original_package_version} now).\n"
                 f"Overriding the version in code with the {version_suffix}."
             )
-            package_version = Version(original_package_version).base_version
+            base_package_version = Version(original_package_version).base_version
             update_version_in__init_py = True
         else:
-            package_version = original_package_version
+            base_package_version = original_package_version
```

Comment:
probably not this one either

Reply:
Ah yes. I just added the line above and did not remove this one :)